frontend/src/lib/components/Cards/InsightCard/InsightCard.tsx

```diff
@@ -128,6 +128,8 @@ export interface InsightCardProps extends Resizeable, React.HTMLAttributes<HTMLD
     insight: InsightModel
     /** id of the dashboard the card is on (when the card is being displayed on a dashboard) **/
     dashboardId?: DashboardType['id']
+    /** Whether the insight has been called to load. */
+    called?: boolean
```

Comment:
Reading this 2 years later I'll have no idea what it means. Can we call it `startedLoading` or something longer yet more descriptive?